interface/forms/gad7/report.php

```diff
@@ -70,4 +84,7 @@ function gad7_report($pid, $encounter, $cols, $id)
     }
 
     print "</tr></table>";
+       /* let user download the pdf */
+       /*  execute interface/forms/gad7/report-as-pdf.php */
+    print '<br> <a  target="_blank" href="' . $genpdf_file_path . '?form_id=' . attr_url($id) . '">' . text($str_generate_pdf) . ' </a><br>';
```

Comment:
looks like the report-as-pdf.php script is missing

Reply:
and this